src/canvas.tsx

```diff
@@ -1,40 +1,41 @@
 import * as THREE from 'three'
 import * as React from 'react'
-import { useRef, useEffect, useMemo, useState, useCallback } from 'react'
+import { useRef, useEffect, useState, useCallback } from 'react'
 import ResizeObserver from 'resize-observer-polyfill'
 import { invalidate, applyProps, render, renderGl, unmountComponentAtNode } from './reconciler'
 
 export type CanvasContext = {
-  canvas?: React.MutableRefObject<any>
-  subscribers: Array<Function>
-  frames: 0
-  aspect: 0
-  gl?: THREE.WebGLRenderer
-  camera?: THREE.Camera
-  scene?: THREE.Scene
-  raycaster?: THREE.Raycaster
-  canvasRect?: DOMRectReadOnly
-  viewport?: { width: number; height: number }
-  size?: { left: number; top: number; width: number; height: number }
   ready: boolean
-  vr: boolean
   manual: boolean
+  vr: boolean
   active: boolean
-  captured: boolean
   invalidateFrameloop: boolean
-  subscribe?: (callback: Function, main: any) => () => any
+  frames: number
+  aspect: number
+  subscribers: Function[]
+  subscribe: (callback: Function) => () => any
   setManual: (takeOverRenderloop: boolean) => any
-  setDefaultCamera: (camera: THREE.Camera) => any
+  setDefaultCamera: (camera: THREE.OrthographicCamera | THREE.PerspectiveCamera) => any
```

Comment:
I think something like instanceof THREE.Camera makes more sense b/c cameras can be custom in three